Core/Config.cpp

```diff
@@ -443,9 +435,7 @@ static ConfigSetting graphicsSettings[] = {
 #endif
 
 	// TODO: Replace these settings with a list of options
-	ConfigSetting("PartialStretch", &g_Config.bPartialStretch, &DefaultPartialStretch, true, true),
-	ConfigSetting("StretchToDisplay", &g_Config.bStretchToDisplay, false, true, true),
-	ConfigSetting("SmallDisplayZoom", &g_Config.iSmallDisplayZoom, 0, true, true),
+	ConfigSetting("SmallDisplayZoom", &g_Config.iSmallDisplayZoom, 2, true, true),
```

Comment:
Hmm, not sure if I'm really a fan of shoving a bunch of different things (including a numeric scale) into the same option.  Especially changing the meaning of "3" from 3x to 1x in an existing option.

Would it make more sense to have "SmallDisplayZoomType" and "SmallDisplayZoomLevel"?

-[Unknown]